components/Tabs/src/MDCTabBar.m

```diff
@@ -256,10 +270,34 @@ - (void)setUnselectedItemTintColor:(UIColor *)unselectedItemTintColor {
   }
 }
 
+- (BOOL)displaysUppercaseTitles {
+  switch (self.titleTextTransform) {
+    case MDCTabBarTextTransformAutomatic:
+      return [MDCTabBar displaysUppercaseTitlesByDefaultForPosition:_barPosition];
+
+    case MDCTabBarTextTransformNone:
+      return NO;
+
+    case MDCTabBarTextTransformUppercase:
+      return YES;
+  }
+}
+
 - (void)setDisplaysUppercaseTitles:(BOOL)displaysUppercaseTitles {
-  _hasDefaultDisplaysUppercaseTitles = NO;
-  _displaysUppercaseTitlesOverride = displaysUppercaseTitles;
-  [self internalSetDisplaysUppercaseTitles:[self computedDisplaysUppercaseTitles]];
+  MDCTabBarTextTransform newTransform = MDCTabBarTextTransformAutomatic;
```

Comment:
Nit: The .automatic value can never be used but is assigned initially.  This could be simplified to:

```objectivec
self.titleTextTransform = 
    displaysUppercaseTitles ? MDCTabBarTextTransformUppercase : MDCTabBarTextTransformNone;
```

Reply:
Done